style.css

```diff
@@ -20,10 +20,64 @@ body {
   background-color: white;
 }
 
-.menu {
+.desktop-menu {
   display: none;
 }
 
```

Comment:
- [ ] OPTIONAL] I advise you to make the HTML scroll behavior smooth, this will make it look more pleasing and feel more enticing 😀.

```
html {
  scroll-behavior: smooth;
}
```